aya_tamimi/task_manager.py

```diff
@@ -0,0 +1,203 @@
+import utils
+from task import Task
+from urgentTask import UrgentTask
+import json
+
+
+tasks = []
```

Comment:
Also, you can use a dictionary instead of a task. It will improve the search functionality.